go/apps/api/routes/v2_ratelimit_limit/400_test.go

```diff
@@ -69,35 +69,6 @@ func TestBadRequests(t *testing.T) {
 		require.NotEmpty(t, res.Body.Meta.RequestId)
 		require.Greater(t, len(res.Body.Error.Errors), 0)
 	})
-
-	// Uncomment and adapt these tests if needed
```

Comment:
why did we comment this out?

Reply:
added back, im not sure why we ever commented it out, mybe due to the checks for the title or something that failed?

I also see res.body.instance which isn't a thing